apps/penxle.com/src/lib/tiptap/components/TiptapFloatingMenu.svelte

```diff
@@ -47,12 +47,14 @@
   };
 
   onMount(() => {
-    editor.registerPlugin(
-      new Plugin({
-        key: new PluginKey(key),
-        view: () => ({ update }),
-      }),
-    );
+    if (editor && !editor.isDestroyed) {
```

Comment:
P3: optional chain 으로 하는 게 적절할 것 같아요!

```suggestion
    if (editor?.isDestroyed) {
```